composer.json

```diff
@@ -9,7 +9,7 @@
     ],
     "minimum-stability": "stable",
     "require-dev": {
-        "phpunit/phpunit": "3.7.*",
+        "phpunit/phpunit": "4.6.*",
```

Comment:
I have both locally, like Travis does. This is OT for this PR though. Step by step.